python/dolfinx_contact/unbiased/nitsche_unbiased.py

```diff
@@ -130,11 +130,15 @@ def nitsche_unbiased(ufl_form: ufl.Form, u: _fem.Function, markers: list[_cpp.me
     # Custom assembly
     # create contact class
     with _common.Timer("~Contact: Init"):
+        mode = dolfinx_contact.cpp.ContactMode.Raytracing
+        mode = dolfinx_contact.cpp.ContactMode.ClosestPoint
```

Comment:
Should the mode be input to nitsche_unbiased?

Reply:
Eventually. I'll first get everything working. Currently im at a debugging stage